src/main/java/practice/StreamPractice.java

```diff
@@ -12,9 +17,18 @@
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
+     * [1,23,4] [1,2,3] -- min && min % 2 == 0
+     * [1,23,4,1,2,3] -- flatmap
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(n -> n.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
Optional. But looks better as for me 
```suggestion
                .flatMap(n -> Arrays.stream(n.split(",")))
```